From 2afafe03d11c3de7ee4aef94a8d37982b88d621d Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 21 Mar 2016 18:31:39 +0200
Subject: [PATCH 0971/1240] tools: a3700: Fix the DDR size and bus width in XDB
 script

- Fix the DDR size (512MB) and bus width (16b) in A3700
  DDR init script for XDB

Change-Id: I4fa9a1e2394322b2e48fcd5ebd568f75e836d1fa
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28423
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 tools/marvell/debugger_scripts/XDB/a3700_ddr.xdb | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tools/marvell/debugger_scripts/XDB/a3700_ddr.xdb b/tools/marvell/debugger_scripts/XDB/a3700_ddr.xdb
index f2749e0..2e1f1f9 100644
--- a/tools/marvell/debugger_scripts/XDB/a3700_ddr.xdb
+++ b/tools/marvell/debugger_scripts/XDB/a3700_ddr.xdb
@@ -83,6 +83,7 @@ SET VALUE /SIZE=LONG CORE:1(0xD0018010)=0x000007AA
 !+++++++++++++++++++++++++++++++++++++++++++++++++
 ! Running DDR initialization in 600MHz mode
 !+++++++++++++++++++++++++++++++++++++++++++++++++
+! DDR MAC init
 SET VALUE /SIZE=LONG CORE:1(0xD0014008)=0x00404500
 SET VALUE /SIZE=LONG CORE:1(0xD0002000)=0x00010000
 SET VALUE /SIZE=LONG CORE:1(0xD0002004)=0x00000000
@@ -91,7 +92,7 @@ SET VALUE /SIZE=LONG CORE:1(0xD0000344)=0x100000aa
 SET VALUE /SIZE=LONG CORE:1(0xD0000310)=0x200000
 SET VALUE /SIZE=LONG CORE:1(0xD0000304)=0x0
 SET VALUE /SIZE=LONG CORE:1(0xD0000308)=0x0
-SET VALUE /SIZE=LONG CORE:1(0xD0000200)=0xe0001
+SET VALUE /SIZE=LONG CORE:1(0xD0000200)=0xD0001
 SET VALUE /SIZE=LONG CORE:1(0xD0000204)=0x0
 SET VALUE /SIZE=LONG CORE:1(0xD0000220)=0x13020532
 SET VALUE /SIZE=LONG CORE:1(0xD0000044)=0x30200
@@ -125,6 +126,8 @@ SET VALUE /SIZE=LONG CORE:1(0xD00003c4)=0x0
 SET VALUE /SIZE=LONG CORE:1(0xD00003dc)=0x81239
 SET VALUE /SIZE=LONG CORE:1(0xD00002c8)=0x0
 SET VALUE /SIZE=LONG CORE:1(0xD0000064)=0x0006
+SET VALUE /SIZE=LONG CORE:1(0xD0000044)=0x00030200
+! DDR PHY init
 SET VALUE /SIZE=LONG CORE:1(0xD0001004)=0x10077779
 SET VALUE /SIZE=LONG CORE:1(0xD0001008)=0x1ff00770
 SET VALUE /SIZE=LONG CORE:1(0xD000100c)=0x3f03fc77
-- 
1.9.1

